paddle/framework/operator.cc

```diff
@@ -13,8 +13,6 @@ See the License for the specific language governing permissions and
 limitations under the License. */
 
 #include "paddle/framework/operator.h"
-#include <algorithm>
-#include "paddle/framework/op_registry.h"
```

Comment:
Why change these lines. If it is not necessary, please leave them unchanged.

Reply:
`paddle/framework/op_registry.h` is not used in this file, keep `#include <algorithm>`